src/app/pages/storage-old/volumes/volumes-list/volumes-list-table-config.ts

```diff
@@ -83,7 +82,6 @@ export class VolumesListTableConfig implements EntityTableConfig {
   // TODO: Unused?
```

Comment:
I see, it's used, I think we can remove that comment

Reply:
It's only read from, but never written to.